styles.css

```diff
@@ -18,61 +15,61 @@
     margin: 0;
     list-style: none;
   }
-  
+
   .img-header {
     flex-grow: 2;
   }
-  
+
   .ic-menu {
     flex-grow: 2;
   }
-  
+
   .flex-main {
     display: flex;
     flex-flow: column wrap;
     gap: 15px;
   }
-  
+
   .name {
     /* Charlie Giorgania */
     font-family: 'Playfair Display', serif;
     font-style: normal;
     font-weight: 400;
     font-size: 50px;
     line-height: 100%;
-    color: #1A2236;
+    color: #1a2236;
     margin-top: 30px;
     margin-bottom: 10px;
     margin-left: 15px;
     width: 80%;
   }
-  
+
   .greetings {
     font-family: 'Poppins', sans-serif;
     font-style: normal;
     font-weight: 400;
     font-size: 14px;
     line-height: 21px;
     letter-spacing: -0.01em;
-    color: #1A2236;
+    color: #1a2236;
     margin-left: 20px;
     margin-right: 20px;
     display: flex;
     flex-direction: column;
     gap: 10px;
   }
-  
+
   .button-collaboration {
     width: 95%;
     align-self: center;
-    background: #1A2236;
-    color: #FFFFFF;
+    background: #1a2236;
+    color: #fff;
     padding: 10px;
   }
-  
+
```

Comment:
- [ ] Kindly make sure the typography is the same as the one on the provided Figma. Please adjust the same `font-weight` of your text.
- [ ]  KIndly make sure the border of this button is the same as the one on the provided Figma design.

    _**Hint: `border: 1px hidden`**_

     - ### YOUR DESIGN
     
     ![image](https://user-images.githubusercontent.com/87197412/219386549-7be85dc8-57eb-4bf9-a59a-99dfee5686a2.png)

     - ### FIGMA DESIGN
     
    ![image](https://user-images.githubusercontent.com/87197412/219386617-5dd5a053-ef7d-4328-ae46-be1acb45b595.png)